app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
```

Comment:
```suggestion
  attr_reader :remember_token
```

sau khi sửa thì test lại xem chạy được chức năng remember k nhé

Reply:
![screenshot from 2019-02-18 11-22-35](https://user-images.githubusercontent.com/46311026/52927917-8529b400-336f-11e9-870a-77d39f3d7472.png)
Ồ có luôn ạ